app/controllers/posts_controller.rb

```diff
@@ -101,6 +102,14 @@ def auth_subscriber
   end
 
   def post_values
-    params.require(:post).permit(:title, :body, :saved, :is_drafted, :closed)
+    params.require(:post).permit(:title, :body, :saved, :is_drafted, :closed, :community_id)
+  end
+
+  def find_my_communities
+    @subscriptions = Subscription.where(account_id: current_account.id)
+    @my_communities = []
+    @subscriptions.each do |subscription|
+      @my_communities << Community.find(subscription.community_id)
```

Comment:
This could trigger a lot of queries depending on the number of subscriptions. Can you reduce this to just one query at the most? @vasanthakumar-a @tusharravindran

Reply:
resolved